src/components/header.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import {Colors} from '../styles/colors';
+
+import {View, Text, StyleSheet, StatusBar} from 'react-native';
```

Comment:
```suggestion
import { View, Text, StyleSheet, StatusBar } from 'react-native';
```
(I don't know if prettier did work here)